homeassistant/components/climate/eq3btsmart.py

```diff
@@ -21,8 +22,6 @@
 _LOGGER = logging.getLogger(__name__)
 
 STATE_BOOST = 'boost'
```

Comment:
We should remove this too.

Reply:
Longer term, definitely. I'm hesitant to do this now as i don't have one of these devices to test against and don't want to remove functionality.